tfhe/src/shortint/parameters/compact_public_key_only/p_fail_2_minus_64/ks_pbs.rs

```diff
@@ -6,13 +6,26 @@ use crate::shortint::parameters::{
     MessageModulus,
 };
 
+/// This parameter set should be used when doing zk proof of public key encryption
 pub const PARAM_PKE_MESSAGE_2_CARRY_2_KS_PBS_TUNIFORM_2M64: CompactPublicKeyEncryptionParameters =
     CompactPublicKeyEncryptionParameters {
-        encryption_lwe_dimension: LweDimension(1024),
-        encryption_noise_distribution: DynamicDistribution::new_t_uniform(42),
+        encryption_lwe_dimension: LweDimension(2048),
```

Comment:
I'll check if the keyswitching parameters need an update as well

Reply:
looks like the keyswitch to big key needs a small update, I'll give you the info to update and have legacy constants there as well